### ISSUE TITLE
Руна преподнесения перестала адекватно работать

### ISSUE THREAD

**Issue by Aviat9181** [frosty-dev/ss14-feedback#133]
1. Иногда(почти в половине случаев) руна преподнесения при попытке вербовки космонавтика и наличии двух культистов (или более) рядом будет говорить о том, "требуется 2 и более культистов", даже несмотря на наличие искомых культистов

2.Шаги для воспроизведения проблемы:
1. Создайте культ
2. Попытайтесь вербовать космонавтиков

3. Должно работать нормально!

4. нету, мяу

**Comment by Aviu00** [comment]
Может человек с майндшилдом.

**Comment by Aviat9181** [comment]
Нет, его бы просто порвало, а тут нет никакого эффекта(и человек был точно без майндшилда)

**Comment by Sevasstyan** [comment]
У меня последние несколько раундов с культом все было нормально, хз.
